update/upgrade_db.sh

```diff
@@ -73,58 +120,101 @@ fi
 echo "Dumping actual DB"
 TODAY=`date '+%Y%m%d'`
 /usr/bin/pg_dump --host $HOST --port 5432 --username "$USER" --format custom --file "$TODAY""_current_qwat.backup" "$SRCDB"
-#/usr/bin/pg_dump -d "service=$QWATSERVICE" --format custom --file "$TODAY""_current_qwat.backup"
 
-echo "Droping existing qwat_test"
+echo "Dropping existing DB qwat_test"
 /usr/bin/dropdb "$TESTDB" --host $HOST --port 5432 --username "$USER" --no-password
-#/usr/bin/dropdb -d "service=$QWATSERVICETEST"
 
-echo "Creating DB (qwat_test)"
+echo "Creating DB qwat_test"
 /usr/bin/createdb "$TESTDB" --host $HOST --port 5432 --username "$USER" --no-password
-#/usr/bin/createdb -d "service=$QWATSERVICETEST"
 
-echo "Restoring in test DB"
+echo "Restoring current DB in qwat_test"
 /usr/bin/pg_restore --host $HOST --port 5432 --username "$USER" --dbname "$TESTDB" --no-password --single-transaction --exit-on-error "$TODAY""_current_qwat.backup"
-#/usr/bin/pg_restore -d "service=$QWATSERVICETEST" --single-transaction --exit-on-error "$TODAY""_current_qwat.backup"
 
-echo "Applying deltas on $TESTDB:"
+TAB_FILES_POST=()
+echo "Applying deltas on $TESTDB :"
 for f in $DIR/delta/*.sql
```

Comment:
Yes it does. I made some test. But yes, we don't allow file names without last digit. Always: x_x_x